src/scripts/preload_models.py

```
@@ -0,0 +1,77 @@
+        print(
+            f"  Downloading models to {output_dir} (code_formula={with_code_formula}, picture_classifier={with_picture_classifier})..."
+        )
+        download_models(
+            output_dir=output_dir,
+            with_code_formula=with_code_formula,
+            with_picture_classifier=with_picture_classifier,
+        )
+        print("  ✓ docling models ready")
+    except ImportError:
+        print("  ✗ docling not installed, skipping")
+    except Exception as e:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)